plugins/modules/ec2_imagebuilder_component.py

```diff
@@ -0,0 +1,399 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Tom Wright (@tomwwright)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: ec2_imagebuilder_component
+short_description: Manage an EC2 Image Builder component
+description:
+    - Manage an EC2 Image Builder component. See U(https://docs.aws.amazon.com/imagebuilder/latest/userguide/) for information about EC2 Image Builder.
+version_added: "2.10"
+requirements: [ boto3 ]
+author: "Tom Wright (@tomwwright)"
+options:
+  change_description:
+    type: str
+    description: ''
```

Comment:
Please add descriptions, going back through and working out what these are supposed to be involves a certain amount of detective work and is much easier if the author adds them in the first place.